util/conversion/conversion.go

```diff
@@ -164,6 +164,7 @@ type FuzzTestFuncInput struct {
 	HubAfterMutation func(conversion.Hub)
 
 	Spoke                      conversion.Convertible
+	SpokeBeforeMutation        func(convertible conversion.Convertible)
```

Comment:
How will this be set?

Reply:
Ah, saw the linked PR now: https://github.com/kubernetes-sigs/cluster-api-provider-aws/pull/2425/files